tests/models/donut/test_processing_donut.py

```diff
@@ -19,12 +19,9 @@
 from transformers import DonutProcessor
 
 
-DONUT_PRETRAINED_MODEL_NAME = "naver-clova-ix/donut-base"
-
-
 class DonutProcessorTest(unittest.TestCase):
     def setUp(self):
-        self.processor = DonutProcessor.from_pretrained(DONUT_PRETRAINED_MODEL_NAME)
+        self.processor = DonutProcessor.from_pretrained("naver-clova-ix/donut-base")
```

Comment:
for this one an dthe 3 others we don't use self.from_pretrained_id ?

Reply:
ah yes woops thanks!